frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -377,27 +381,31 @@ export const GRAPH_DISPLAY_VALUES_SETTINGS = {
     getHidden: (series, vizSettings) =>
       vizSettings["stackable.stack_type"] === "normalized",
     default: false,
+    inline: true,
+    marginBottom: "1rem",
   },
   "graph.label_value_frequency": {
     section: t`Display`,
     title: t`Values to show`,
-    widget: "radio",
+    widget: "segmentedControl",
     getHidden: (series, vizSettings) =>
       vizSettings["graph.show_values"] !== true ||
       vizSettings["stackable.stack_type"] === "normalized",
     props: {
       options: [
-        { name: t`As many as can fit nicely`, value: "fit" },
+        { name: t`Some`, value: "fit" },
         { name: t`All`, value: "all" },
       ],
+      variant: "fill-background",
```

Comment:
This is a question for @cdeweyx and @naatz-co. Are we using this new style as only a single style for all segmented controls moving forward or are we expected to still use the 2 styles concurrently?

See the screenshot below which has both styles, I think it looks a little inconsistent.
![image](https://user-images.githubusercontent.com/1937582/192722954-97969f0e-a04b-4889-97dc-17c963ceb581.png)

Here's how it'd look like if every segmented control has the same style `fill-background`
![image](https://user-images.githubusercontent.com/1937582/192723127-5b6865c6-7a80-4fee-a8a7-6dd63f866a0d.png)

What do you guys think? 

The answer to this question could influence this change since if we're aiming to have only a single style we could have only changed the default style for all `SegmentedControl` here.
https://github.com/metabase/metabase/blob/6a9f2f1d9d41c9b8476d59966f3dc8f28a4b7215/frontend/src/metabase/components/SegmentedControl.jsx#L43

Reply:
That seems preferable to me. Nice catch. @naatz-co WDYT?

Reply:
the other thing that I changed was the inactiveText color. If we default to filled, we may want to default that to `text-dark` as well

Reply:
ahh yep 👍 what I'm saying is that instead of inlining style in multiple places, if we can only have a single style, we can just make changes just to the `SegmentedControl`.

Reply:
@npfitz Since Nathalia is out and it looks like we have consensus in Notion, let's do this. No blockers on the product side. 👍

Reply:
@naatz-co is out sick, so I'll answer — we should use `fill-background` for this component globally now. Great catch.

Reply:
Works for me! Thanks all